ui/raidboss/data/06-ew/raid/p1n.ts

```diff
@@ -132,6 +132,19 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'en',
+      'replaceText': {
+        'Gaoler\'s Flail Left/Gaoler\'s Flail Right': 'Gaoler\'s Flail Left/Right',
+        'Gaoler\'s Flail Right/Gaoler\'s Flail Left': 'Gaoler\'s Flail Right/Left',
+        'Hot Spell/Cold Spell': 'Hot/Cold',
```

Comment:
```suggestion
        'Hot Spell/Cold Spell': 'Hot/Cold Spell',
```

Small change to prefer the full name unless it's really long.